configs/components/openssl.rb

```diff
@@ -79,16 +79,15 @@
     pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-gcc-5.2.0-11.aix#{platform.os_version}.ppc.rpm"
   elsif platform.is_macos?
     pkg.build_requires 'makedepend'
-  elsif platform.is_linux?
-    unless platform.is_fedora? && platform.os_version.delete('f').to_i >= 26
-      # TODO: pdk had this for all linux platforms, but agent didn't - necessary?
-      pkg.build_requires 'pl-binutils'
-    end
+  elsif platform.is_cross_compiled_linux?
+    pkg.build_requires 'pl-binutils'
     pkg.build_requires 'pl-gcc'
```

Comment:
I'm a bit surprised this was working previously - typically the pl-build-tools package names we've had to specify for cross-compiled builds included the architecture in them, eg, pl-gcc-<arch>. But if that dependency isn't met I'd expect things to blow up quickly.